### PR TITLE
Quick routing updates

### PR DESCRIPTION
**Pull request by daniel-panhead**
Tiny routing fixes

Heavily considering migrating the whole site to Nuxt

### CODE REVIEW

**Comment by daniel-panhead** [comment]
Actually nvm I'm too lazy let's just stick with client side routing

**Comment by daniel-panhead** [comment]
Lazy loading should be enough

**Comment by daniel-panhead** [comment]
okay actually idk but that's an issue for another pr

File: src/router/index.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-import HomeView from '@/views/HomeView.vue'
-import IdeasView from '@/views/IdeasView.vue'
-import TeamView from '@/views/TeamView.vue'
-import ELSIView from '@/views/ELSIView.vue'
-import GroupAView from '@/views/enzymosome/GroupAView.vue'
+const HomeView = () => import('@/views/HomeView.vue')
+const IdeasView = () => import('@/views/IdeasView.vue')
+const TeamView = () => import('@/views/TeamView.vue')
+const ELSIView = () => import('@/views/ELSIView.vue')
+const GroupAView = () => import('@/views/enzymosome/GroupAView.vue')
```
Comment (IcedTea2K) [inline src/router/index.ts:6]: Yeah, I was gonna ask you about this. But it seems like you figured out the lazy loading so lesgoooo! 💯

**Review by IcedTea2K — APPROVED** [review]
LGTM! 👍

**Pull request merged**